scene/main/node.cpp

```diff
@@ -1498,8 +1526,64 @@ void Node::_set_owner_nocheck(Node *p_owner) {
 	data.OW = data.owner->data.owned.back();
 }
 
+void Node::_release_unique_name_in_owner() {
+	ERR_FAIL_COND(!data.owner); // Sanity checl
+	StringName key = StringName(UNIQUE_NODE_PREFIX + data.name.operator String());
+	Node **which = data.owner->data.owned_unique_nodes.getptr(key);
+	if (which == nullptr || *which != this) {
```

Comment:
```suggestion
	if (!which || *which != this) {
```
idk if we have that in style guide though.

Reply:
Note that you made a mistake here (it should be `if (which`). I think this verbosity is good for code like this as it helps differentiate from booleans and it also makes the intention more intuitive to avoid mistakes.

Reply:
This does not mean I believe there should be a style guide for this to be honest, contributors generally have their own style and I think its ok code differs to some amount between them as long as the general thing is the same.

Reply:
> Note that you made a mistake here (it should be if (which)

`if (which)` is equivalent of `if (which != nullptr)`, so my suggestion was correct :v